src/hooks/screen/useMessageHook.ts

```diff
@@ -115,12 +115,12 @@ function useMessageHook(): UseMessageHook {
 
   const onOpenMediaPreview = (medias, index, navigation) => {
     if (medias.find((media) => media.type === 'video')) {
-      navigation.push('VideoViewer', {
+      navigation.navigate('VideoViewer', {
         title: 'Video',
         url: medias[index].video_path
       });
     } else {
-      navigation.push('ImageViewer', {
+      navigation.navigate('ImageViewer', {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `navigation.push` to `navigation.navigate` could potentially cause issues if the user is already on the 'VideoViewer' or 'ImageViewer' screen and tries to open another media preview. The `navigate` function will not push a new screen onto the stack if the screen is already in the stack, unlike the `push` function which always adds a new screen to the stack. If this behavior is intended, then it's fine. Otherwise, you might want to reconsider using `navigate`.

```diff
- navigation.navigate('VideoViewer', {
+ navigation.push('VideoViewer', {
     title: 'Video',
     url: medias[index].video_path
 });

- navigation.navigate('ImageViewer', {
+ navigation.push('ImageViewer', {
     title: 'Photo',
     index,
     images: medias
});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
@GarryLaly any concern why using navigate instead of push?

Reply:
@fajarism ikut ke commit, tadi buat testing glitch ketika back, ternyata bukan ini solusinya. Sudah di revert pakai .push lagi ya mas